.github/CONTRIBUTING.md

```diff
@@ -42,4 +44,4 @@ If there are any failures then the automated tests fail.
 These tests are run both with the latest available version of Nextflow and also the minimum required version that is stated in the pipeline code.
 
 ## Getting help
-For further information/help, please consult the [nf-core/rnaseq documentation](https://github.com/nf-core/rnaseq#documentation) and don't hesitate to get in touch on [Gitter](https://gitter.im/nf-core/Lobby)
+For further information/help, please consult the [nf-core/rnaseq documentation](https://github.com/nf-core/rnaseq#documentation) and don't hesitate to get in touch on the pipeline channel on [Slack](https://nfcore.slack.com/channels/rnaseq).
```

Comment:
```suggestion
For further information/help, please consult the [nf-core/rnaseq documentation](https://github.com/nf-core/rnaseq#documentation) and don't hesitate to get in touch on the pipeline channel on [Slack](https://nfcore.slack.com/channels/rnaseq).
```
```suggestion
For further information/help, please consult the [nf-core/rnaseq documentation](https://nf-co.re/rnaseq#documentation) and don't hesitate to get in touch on the pipeline channel on [Slack](https://nfcore.slack.com/channels/rnaseq).
```
I'm guessing we can even link the nf-core website for docs, no?

Reply:
Yup we can 👍

Reply:
So why did you resolved the conversation, if you did not accept the changes?

Reply:
But I applied your suggestion? 😕

Reply:
I guess there was an issue with the suggestions from my side, I'll unresolve this conversation so we can commit again

Reply:
Odd, it's there already 😕

Reply:
Now it is, I commited it here: https://github.com/nf-core/rnaseq/pull/300/commits/7823546fe874c962dc29cab85d0c21862aaf6e27